packages/kit/src/core/prerender/prerender.js

```diff
@@ -285,7 +285,9 @@ export async function prerender({ config, entries, files, log }) {
 							location: resolved
 						});
 
-						prerendered.paths.push(normalize_path(decoded, 'never'));
+						prerendered.paths.push(
+							normalize_path(decoded, file.endsWith('/index.html') ? 'always' : 'never')
```

Comment:
Why do we need to handle `/index.html` differently? Do we need to handle `/index.html` differently in all cases, so it should be moved into `normalize_path`?

Reply:
I started writing an explanation of this but it just made me realise we can simplify the implementation... one sec